<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Deepika" failures="1" tests="3" name="ChromeTest" time="114.834" errors="0" timestamp="2023-07-07T01:07:43 IST">
  <testcase classname="lambdatest.TestScenarios" name="testScenario1" time="1.336">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [LambdaTest] but found [Selenium Grid Online | Run Selenium Test On Cloud]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [LambdaTest] but found [Selenium Grid Online | Run Selenium Test On Cloud]
at lambdatest.TestScenarios.testScenario1(TestScenarios.java:121)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- testScenario1 -->
  <testcase classname="lambdatest.TestScenarios" name="testScenario2" time="5.819"/>
  <testcase classname="lambdatest.TestScenarios" name="testScenario3" time="7.041"/>
</testsuite> <!-- ChromeTest -->
